website/routes.py

```diff
@@ -85,7 +85,7 @@ def get_globals():
     location = geolite2.reader().get(request.remote_addr) if request.remote_addr else None
     if request.host_url != settings.DOMAIN:
         try:
-            inst_id = Institution.objects.get(domains__icontains=[request.host])._id
+            inst_id = Institution.objects.get(domains__icontains=request.host, is_deleted=False)._id
```

Comment:
@Johnetordoff, good improvement here.  I've added some notes to the JIRA ticket; not sure if all the behavior has been addressed.